rules/removedelete.mutations.js

```diff
@@ -58,12 +58,43 @@ const checkWrongName = (text, node, messages, name) => {
   }
 };
 
+function getMutationNames(ast) {
```

Comment:
Could you place this function in a new file in the "lib" folder, maybe helpers.js or mutations.js, whatever you think is best.
It's now duplicated in two places and if we are gonna have more rules for mutations we should probably use this there as well.